install.js

```diff
@@ -114,30 +115,30 @@ var configItems = [
     },
     default: 'N'
   },
-  {
-    name: 'smtpService',
-    type: 'string',
-    description: "Which SMTP service (if any) will be used? (see https://github.com/andris9/nodemailer-wellknown#supported-services for a list of supported services.)",
-    default: 'none'
-  },
-  {
-    name: 'smtpUsername',
-    type: 'string',
-    description: "SMTP username",
-    default: ''
-  },
-  {
-    name: 'smtpPassword',
-    type: 'string',
-    description: "SMTP password",
-    hidden: true
-  },
-  {
-    name: 'fromAddress',
-    type: 'string',
-    description: "Sender email address",
-    default: ''
-  },
+//  {
+//    name: 'smtpService',
```

Comment:
I think these should be left in the manual install, otherwise we'll have to document how to enable email notifications for password resets.  Can these be uncommented but defaulted/bypassed for the Vagrant script?